src/main/webapp/util/ReportConstants.js

```diff
@@ -10,15 +10,26 @@ export const SORT_NAME_INDEX = {
   'awardNumber': 8
 };
 
-export const STYLES = {
+export const CATEGORY_SORT_NAME_INDEX = {
+  "projectKey": 0,
+  "summary": 1,
+  "status": 2
+};
+
+export const STYLES_FUNDING_REPORTS = {
   issueTypeWidth: '96px',
   projectKeyWidth: '109px',
   titleWidth: '280px',
   statusWidth: '98px',
   pisWidth: '80px',
   protocolWidth: '94px',
   fundingNameWidth: '124px',
-  generalWidth: '136px'
+  generalWidth: '136px',
 };
 
-
+export const STYLES_REVIEW_CATEGORIES = {
```

Comment:
Another way of doing this so you don't have such long and unwieldy names, make one style object with sub objects:
```
export const styles = {
  fundingReport: { ... },
  reviewCategories: { ... }
}
```